### DISCUSSION TITLE
Request to Check Possibility to Support for Gatling Report Genration from Simulation Log

### DISCUSSION THREAD

**Discussion by Ntm2024** [ludeknovy/jtl-reporter#286] [Ideas]
**Sample Simulation Log File**
[simulation.log](https://github.com/user-attachments/files/15520893/simulation.log)

**Comment by ludeknovy** [comment]
Hi @Ntm2024. I am sorry, but I do not plan to add support of simulation log files processings. The preferred way would be parse the simulation log fine into JTL/CSV format locally, and then uploading it to the JtlReporter.

**Reply by Ntm2024** [reply]
@ludeknovy Thank you for the feedback. I'll try the mentioned approach.

**Reply by ludeknovy** [reply]
If you manage to make it work, please share it here. Thanks.
